srcpkgs/python3-coverage/template

```diff
@@ -1,17 +1,38 @@
 # Template file for 'python3-coverage'
 pkgname=python3-coverage
-version=5.0.3
-revision=5
-build_style=python3-module
-hostmakedepends="python3-setuptools"
+version=7.2.7
+revision=1
+build_style=python3-pep517
+hostmakedepends="python3-wheel"
 makedepends="python3-devel"
-depends="python3-setuptools"
-checkdepends="python3-pytest-xdist python3-flaky python3-unittest-mixins"
+checkdepends="$depends python3-pytest-xdist python3-flaky
+ python3-hypothesis"
 short_desc="Code coverage testing for Python3"
 maintainer="Orphaned <orphan@voidlinux.org>"
 license="Apache-2.0"
-homepage="https://bitbucket.org/ned/coveragepy"
+homepage="https://github.com/nedbat/coveragepy"
+changelog="https://github.com/nedbat/coveragepy/raw/master/CHANGES.rst"
```

Comment:
I think this could be fixed by telling curl within xchangelog to follow redirects when getting the mimetype

Reply:
I guess so, yeah, though I like to use the direct link anyway for consistency's sake (and because there's no good reason to add an extra redirect for everybody opening the URL).

Reply:
Fair enough, thanks for the comments.

I just right-clicked and used "copy link" to get the link, and my xchangelog uses `curl -sL` to download so it works and I didn't realize this was a redirect. The server changes so it's not even a "free" redirect.

OTOH I'd worry about which URL is the most stable one of the two (for some reason gh itself links via the first one and then redirects) but I guess it's unlikely that either one changes.

I'll review everything and update all the changelog links (on the bright side, I think I got text changelogs for everything).

Moreover, I want to work out a little bit more on the templates that use a venv to check, to see if a similar workaround as in #44268 might work instead of a custom `do_check()`. Keep tuned.

Reply:
All changelog entries are replaced by the direct link.